### PR TITLE
UIU-2732: Disable 'Claimed return' button after click to prevent multiple submissions

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
## Purpose
Disable 'Claimed return' button after click to prevent multiple submissions.

## Refs
https://issues.folio.org/browse/UIU-2732

### CODE REVIEW

**Review by zburke — CHANGES_REQUESTED** [review]
Sorry, it's really hard to figure out what's actually happening here given the very generic "Fix problem" in the PR description and "toggleButton" variables in the code. Additionally, new functionality needs new tests. Maybe you need a more extensive mock of the Modal opened by LoanActionDialog so you can validate whether its buttons are enabled based on the `inProgress` prop you're passing through?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 * Fix problem with displaying suspended claim returned fees/fines. Refs UIU-2726.
 * Adjust permissions in order to see the link to open request queue. Fixes UIU-2661.
 * Fix systems error when attempting to create a new fee/fine if logged in user isn't logged into a service point. Refs UIU-2728.
+* Fix problem with Claimed return button. Refs UIU-2732.
```
[the remark this answers is not in the document]
Comment (Dmitriy-Litvinenko) [reply CHANGELOG.md:9]: Fixed

File: src/components/Wrappers/withClaimReturned.js [hunk 2]
```diff
@@ -96,11 +97,16 @@ const withClaimReturned = WrappedComponent => class withClaimReturnedComponent e
     this.setState({ claimReturnedDialogOpen: false });
   }
 
+  toggleButton = (inProgress) => {
+    this.setState({ inProgress });
+  };
+
```
[the remark this answers is not in the document]
Comment (Dmitriy-Litvinenko) [reply src/components/Wrappers/withClaimReturned.js:103]: Fixed

**Comment by Dmitriy-Litvinenko** [comment]
@zburke Could you please re-review?
@mkuklis Could you please review?

**Review by zburke — COMMENTED** [review]
The better variable names make this much easier to follow. Thank you! But now that I see what's going on, I have a question: `<ModalContent>` seems like a generic component. Does it really need to know about `claimedReturnInProgress` and `declarationInProgress`? It feels instead like there should be a generic `isInProgress` prop so we don't have to add a new prop to it every time we want to call it from a new component, i.e.
```
const isConfirmButtonDisabled = !additionalInfo || claimReturnedInProgress || declarationInProgress;
```
can be rewritten as 
```
const isConfirmButtonDisabled = !additionalInfo || isInProgress;
```
Good idea, bad idea?

**Review by zburke — APPROVED** [review]
Sorry for the run-around from generic props to implementation-specific props and back again. Now that I have a solid understanding of how this works, and with UIU-2736 in the picture, this looks great!

**Pull request merged**